docs/OFFCHAIN_API.md

```diff
@@ -0,0 +1,109 @@
+# SunflowerLand Offchain Open API
+
+This documentation communicates how to query SunflowerLand's offchain API.
+
+This information is relevant for developers who are looking to build custom tooling around SunflowerLand's most current, offchain data.
+
+### Rate-Limits
+
+There are some rate-limits imposed while querying this API. For best results, query these batch APIs at a 15 second interval. Querying at higher intervals will amost certainly result in failure response: `HTTP 429 Too Many Requests`.
+
+## Batch Farm Data
+
+Query current offchain state for a batch amount of farms.
+
+### URL:
+
+```
+POST https://api.sunflower-land.com/community/getFarms
+```
+
+### Request Body:
+
+A unique list of farmIds, no duplicates. You can query up to 500 farms per request.
```

Comment:
Is up to 100 afaik and I think it's the same for the Bumpkins one 👍

Reply:
@vitt0 awesome, thanks for letting me know. I will do some Discord research to confirm and update accordingly.